config/general.php

```diff
@@ -41,7 +41,7 @@
     // Dev environment settings
     'dev' => [
         // Base site URL
-        'siteUrl' => null,
+        'siteUrl' => getenv('SITE_URL'),
```

Comment:
This will necessitate adding a new local environment variable but it also means we can properly preview things locally again.